how-we-work/all-hands-meeting.html.md

```diff
@@ -19,33 +44,17 @@ All Hands events need some work from the meeting organizer both to prepare the m
 to be sure everyone is aware of the meeting and what to expect from it. If you are in charge of
 organizing the All Hands, simply follow these tips to make it a success!
 
-### One day before the meeting
-
+- **Check the calendar event.** We have a shared Nebulab calendar that everyone has joined during
+  their onboarding. In this calendar we have all the meetings events already scheduled. Just be
+  sure information are accurate in the calendar event or change the meeting date if needed.
 - **Prepare the agenda.** Prepare the agenda in Tadum, listing any topics that will be discussed
   during the event. This is important so everyone is allowed to take a look and ask for
   clarifications about topics or propose additional topics.
 - **Notify the team.** Once the agenda is ready, send a message into the #all-hands Slack channel to
-  let everyone know that the meeting will be tomorrow and that they can read the agenda in advance.
-
-### The day of the meeting
-
-- **Notify the team.** Send another message in the #all-hands Slack channel to let everyone know the
-  meeting time. This message can be used to remind people of important things like why attendance
-  (and being on time) are important, how to join the meeting and how to ask questions.
-
-#### After the meeting
-
-After the meeting the organizer needs to close the agenda on Tadum and add any topics to be
-discussed in the next meeting.
-
-### Pro-tip: Slack reminders
-
-Organizers can use the following Slack reminders to remind them of any actions they need to take
-to organize All Hands meetings:
-
-- `/remind me to "Send Slack reminder for All Hands (meeting is tomorrow)" next Thursday 15pm every 4 weeks`
-- `/remind me to "Send Slack reminder for All Hands (meeting is Today)" next Friday 10am every 4 weeks`
-- `/remind me to "Close All Hands agenda" next Friday 17pm every 4 weeks`
-
-Please note that these commands need to be run **no later than Thursday during the the week of the
-meeting**.
+  let everyone know when we'll have the meeting and that they can read the agenda in advance. It's
+  also a good practice to notify the event some hour before it starts. This is also a good moment
+  to remind people of important things like why attendance (and being on time) are important,
```

Comment:
The verb here should be singular, "is" important. I would use a singular even if "and being on time" wasn't in parenthesis, but don't ask me why... It just reads better that way when the subjects are verbs.